docs/upgrade.adoc

```diff
@@ -0,0 +1,73 @@
+= React4XP - Upgrade
+:toc: right
+
+We have tried to simplify the build system and limit the number of configurable combinations to less than the number of atoms in the universe :)
```

Comment:
Sure, but you just removed comments like this above?

Reply:
Yes, I figured you would see that one. Feel free to write something more productive.

Reply:
Fixed